azavea/raster-foundry
# Feature/cmb/move footprint field

This removes the footprint geometry relation and adds a footprint field directly to scenes since that is most convenient and a scene will only ever have a single footprint.

## Testing
- ensure that `/api/scenes/` works correctly with dummy data
- run tests
